poolmanager/manager.go

```diff
@@ -296,20 +296,20 @@ func (lbg *lbGroup) target(ts time.Time, target int64) {
 
 		if desiredScale > lbg.target_capacity {
 			// We still need additional capacity
-			wanted := math.Min(math.Ceil(float64(target-lbg.target_capacity)/cp.CAPACITY_PER_RUNNER), LARGEST_REQUEST_AT_ONCE)
+			wanted := math.Min(math.Ceil(float64(target-lbg.target_capacity)/controlplane.CAPACITY_PER_RUNNER), LARGEST_REQUEST_AT_ONCE)
 			asked_for, err := lbg.cp.ProvisionRunners(lbg.Id(), int(wanted)) // Send the request; they'll show up later
 			if err != nil {
 				// Some kind of error during attempt to scale up
 				logrus.WithError(err).Error("Error occured during attempt to scale up")
 				return
 			}
-			lbg.target_capacity += int64(asked_for) * cp.CAPACITY_PER_RUNNER
+			lbg.target_capacity += int64(asked_for) * controlplane.CAPACITY_PER_RUNNER
 		}
 
-	} else if desiredScale <= lbg.current_capacity-cp.CAPACITY_PER_RUNNER {
+	} else if desiredScale <= lbg.current_capacity-controlplane.CAPACITY_PER_RUNNER {
```

Comment:
can-we_do_someTHING-About-THIS_too?

Reply:
I'll do a followup pr to clean all this up